### ISSUE TITLE
Command event replaces {player} with caster instead of target

### ISSUE THREAD

**Issue by KiviTheFruit** [Eniripsa96/SkillAPI#492]
Version: SkillAPI Premium 1.90

When using the command event with the {player} variable, then it will always use the casters name and not the name of the targeted player.

**Comment by xxJinxedxx** [comment]
This depends on how you will use the Command mechanic.

**Comment by KiviTheFruit** [comment]
[Heilender Regen.txt](https://github.com/Eniripsa96/SkillAPI/files/3780316/Heilender.Regen.txt)

This Skill should use the command "betterhealing {player} 20" and replace the {player}-variable with all names of allies in a 7 block radius. Instead it will use the command "betterhealing caster 20" once for every nearby ally. Or is there somethin i've done wrong?

**Comment by xxJinxedxx** [comment]
Yes, at this moment the area targeting options can not target the "targets" in that way, meaning you can not have a Command under area targeting unless you want to target the caster.
I have a skill that need to use that same construct and i still haven't thought of a way to bypass that one.
It is not a bug or an issue tho, this is the way it works. Hopefully with the new Kotlin update there will be an improvement on it.

You can join the [discord](https://discord.gg/TR64Fy) if you need any additional help.

**Comment by xxJinxedxx** [comment]
Actually yes you are right, it is an issue. Tbh i haven't paid attention to the description of the mechanic at all..
![Capture](https://user-images.githubusercontent.com/13562136/67776115-8a93de00-fa68-11e9-8c32-53b59a535c47.PNG)

**Comment by KiviTheFruit** [comment]
The area targeter seems to work tho. The message event works as it should, it's just the command event, that doesn't work.
For now i use another plugin, wich will use the commands correctly, so it's not a critical issue for me now.
